website/pages/blog/aws-priv-esc-identity-center.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: 'AWS Identity Center (formerly known as AWS SSO): A Guide to Privilege Escalation and Identity and Access Management of AWS Identity Center'
+tag: security
+date: 2023/05/01
+description: >-
+  AWS Identity Center is one way of managing access to AWS Accounts.  With AWS Identity Center (previously SSO), there exists multiple pathways to privilege escalation.  In this blog post, we cover Identity Center, research into the inner workings of cloud Identity and Access Management related to AWS Identity Center, how to secure AWS Identity Center, and detection with logging and monitoring.  
+author: jsonkao
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+## Overview
+
+AWS Identity Center ([formerly known as AWS SSO](https://docs.aws.amazon.com/singlesignon/latest/userguide/what-is.html#renamed)) is one method of providing and managing cloud access to Amazon Web Services accounts within an Organization, resources in AWS accounts, and applications hosted in AWS.  As with other methods of access to AWS accounts including IAM Roles and Users, there is the potential for misconfiguration and paths to privilege escalation and other security issues within AWS environments.
+
+In this blog, we will cover methods for privilege escalation with AWS Identity Center, access disruption, and other events that could indicate security issues within an AWS environment.  We’ll cover our research and observations on AWS Identity Center from our deep dive and experimentation with AWS Identity Center, with a specific focus on AWS Identity Center and attack vectors introduced by the usage of AWS Identity Center.  There are existing methods of privilege escalation that have been covered and researched by the security community such as `iam:PassRole` , `sts:AssumeRole` , and `iam:SetDefaultPolicyVersion`.
```

Comment:
Maybe a link here to previous research (for `iam:PassRole`, `sts:AssumeRole` and `SetDefaultPolicy`)?

Reply:
Good point, added!

Reply:
Good point!  Added.